src/analysis/normed_space/basic.lean

```diff
@@ -299,13 +296,11 @@ nnreal.eq $ norm_mul a b
 @[simp] lemma nnnorm_pow (a : α) (n : ℕ) : ∥a ^ n∥₊ = ∥a∥₊ ^ n :=
 (nnnorm_hom.to_monoid_hom : α →* ℝ≥0).map_pow a n
 
-@[simp] lemma norm_prod (s : finset β) (f : β → α) :
-  ∥∏ b in s, f b∥ = ∏ b in s, ∥f b∥ :=
-(norm_hom.to_monoid_hom : α →* ℝ).map_prod f s
+lemma list.norm_prod (l : list α) : ∥l.prod∥ = (l.map norm).prod :=
+(norm_hom.to_monoid_hom : α →* ℝ).map_list_prod _
```

Comment:
For another reviewer: is this name better or worse than `norm_list_prod`?

Reply:
I prefer `norm_list_prod` to match `map_list_prod` and friends.

Reply:
But doesn't `list.norm_prod` give you dot notation?

Reply:
Eh yeah, it does. Both are fine, really. If you want to keep it like that, add `protected` at least.